pkg/test/framework/components/echo/sidecar.go

```diff
@@ -32,6 +32,11 @@ type Sidecar interface {
 	Config() (*envoyAdmin.ConfigDump, error)
 	ConfigOrFail(t test.Failer) *envoyAdmin.ConfigDump
 
+	// Config of the Envoy instance with EDS included.
+        // This will be used to verify the EDS configuration in dual stack testing.
```

Comment:
I don't think we should be introspecting XDS config in e2e tests like I assume this is for - we have plenty of unit tests designed for these purposes

Reply:
We did this in our testing to verify the configuration besides the traffic. I am ok to leave those out since we are not planning to verify the configuration at this point.